src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --header-hover-color: #00acdc;
```

Comment:
it would be better to name something like `accent-color` in case you use this variable not only to the hover effect